scom/stackstate_checks/scom/data/conf.yaml.example

```diff
@@ -0,0 +1,13 @@
+init_config:
+    # run every 5 minute
+    min_collection_interval: 300
+
+instances:
+  - hostip: 34.244.241.169
```

Comment:
Will it be always IP or hostname work as well ? If yes, then we should rename `hostip` to `host` and then specify in the comment.